compiler/luci-interpreter/src/kernels/FullyConnected.test.cpp

```diff
@@ -154,6 +193,24 @@ TYPED_TEST(FullyConnectedTest, Simple)
                    });
 }
 
+TEST(FullyConnectedTest, SimpleS4)
+{
+  CheckHybridS4({1, 2}, {4, 2}, {4}, {1, 4},
+                {
+                  1, 3, // batch = 0
+                },
+                {
+                  0, 1,   // unit = 0
+                  0, 0,   // unit = 1
+                  -1, -1, // unit = 2
+                  0, 0,   // unit = 3
+                },
+                {0, 1, 2, 3},
+                {
+                  1.5, 1, 0, 3, // batch = 0
+                });
```

Comment:
this test calls `CheckHybridS4` and does not have any gtest provided check methods like `EXPECTEQ()`. for this our SE analysis tool may complain that there is no actual check. I suggest to move body of `CheckHybridS4` inside here as I only see one call.